src/components/tooltip/tooltip.js

```diff
@@ -148,40 +148,31 @@ const Tooltip = props => {
   const BodyComponent = props.components.BodyComponent || Body;
   const TooltipWrapperComponent =
     props.components.TooltipWrapperComponent || TooltipWrapper;
+
   return (
-    <Manager>
-      <Reference innerRef={childrenRef}>
-        {({ ref }) => (
-          <WrapperComponent {...eventListeners}>
-            <RootRef rootRef={ref}>
-              {React.cloneElement(props.children, { ...childrenProps })}
-            </RootRef>
-          </WrapperComponent>
-        )}
-      </Reference>
+    <React.Fragment>
+      <WrapperComponent {...eventListeners} ref={reference.ref}>
+        {React.cloneElement(props.children, { ...childrenProps })}
+      </WrapperComponent>
       {tooltipIsOpen && (
         <TooltipWrapperComponent>
-          <Popper placement={props.placement} positionFixed={true}>
```

Comment:
I think it was a mistake to have positionFixed set by default anyways, it's not really the behaviour we are looking for